solarkraft/test/e2e/generated/setterHardcoded.ts

```diff
@@ -0,0 +1,6 @@
+// AUTO-GENERATED by setter-populate.sh
+export const SETTER_HEIGHT = 53424
```

Comment:
Sim, do we need to commit this?

Reply:
well, I imagine that we do not push new addresses often, just when the testnet is reset, though the deployment is tried every night. We will debug the tests more often.